src/ManifoldsBase.jl

```diff
@@ -384,69 +384,110 @@ function shortest_geodesic(M::Manifold, x, y, T::AbstractVector)
     return geodesic(M, x, log(M, x, y), T)
 end
 
+abstract type AbstractVectorTransportMethod end
+
+"""
+    ParallelTransport <: AbstractVectorTransportMethod
+
+Specify to use parallel transport as vector transport method within
+[`vector_transport_to`](@ref), [`vector_transport_direction`](@ref) or
+[`vector_transport_along`](@ref). 
+"""
+struct ParallelTransport <: AbstractVectorTransportMethod end
+
+"""
+    ProjectTangent <: AbstractVectorTransportMethod
+
+Specify to use projection onto tangent space as vector transport method within
+[`vector_transport_to`](@ref), [`vector_transport_direction`](@ref) or
+[`vector_transport_along`](@ref). See [`project_tangent`](@ref) for details.
+"""
+struct ProjectTangent <: AbstractVectorTransportMethod end
```

Comment:
I don't really like this name. Firstly it does not specify that it's a transport method and secondly, the same struct could in the future be used for projection-based transport of cotangent vectors (for example). Maybe `ProjectionTransport`, `ProjectTransport` or `ProjectionVectorTransport`?

Reply:
Oh, good remark. I think I am in favour of `ProjectionTransport`, that would then also apply for `CoTVectors`.